modules/core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/SQLInfo.java

```diff
@@ -114,7 +114,7 @@
 
     private final Map<String, Column> copyIdColumnMap;
 
-    protected final Map<String, SQLInfoSelect> selectFragmentById;
+    public final Map<String, SQLInfoSelect> selectFragmentById;
```

Comment:
is this required for test purpose?

Reply:
No it's for `VCSLockManager` which is in a different package.